src/transaction.js

```diff
@@ -267,7 +307,12 @@ Transaction.prototype.toBuffer = function () {
 
   writeVarInt(this.outs.length)
   this.outs.forEach(function (txOut) {
-    writeUInt64(txOut.value)
+    if (!txOut.valueBuffer) {
+      writeUInt64(txOut.value)
+    } else {
+      writeSlice(txOut.valueBuffer)
+    }
```

Comment:
Admittedly, this does suck a little bit.  We almost got away with never needing `>2^53`, but I think this covers the use case sufficiently.

I don't like that it is no longer symmetrical with `fromBuffer`.
Open to ideas on a better solution here.